### PR TITLE
DISCO-1388 return course_uuid in course run serializer

### PR DESCRIPTION
**Pull request by xnick421**

### CODE REVIEW

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -739,6 +739,7 @@ def validate(self, attrs):
 class CourseRunSerializer(MinimalCourseRunSerializer):
     """Serializer for the ``CourseRun`` model."""
     course = serializers.SlugRelatedField(required=True, slug_field='key', queryset=Course.objects.filter_drafts())
+    course_uuid = serializers.ReadOnlyField(source='course.uuid')
```
Comment (Dillon-Dumesnil) [inline course_discovery/apps/api/serializers.py]: Why do we need to add this into the serializer? Can't we just grab this info from off the course object we already serialize as part of the course run?

Comment (xnick421) [reply course_discovery/apps/api/serializers.py]: the `course` field above this line you mean? thats a slug

Comment (xnick421) [reply course_discovery/apps/api/serializers.py]: additionally, the front end is passing a `fields=course_uuid` param, so the payload is very small

Comment (Dillon-Dumesnil) [reply course_discovery/apps/api/serializers.py]: Yeah, I saw that once I saw the frontend PR. Okay. Is this going to have any issues if course isn't set yet? Like is this going to error out or anything with a new course run when it is trying to source `course.uuid`, but `course` isn't set yet.

**Review by Dillon-Dumesnil — APPROVED** [review]

File: course_discovery/apps/api/serializers.py (hunk 1 above)
Comment (xnick421) [reply course_discovery/apps/api/serializers.py]: prob

**Review by mikix — COMMENTED** [review]
Is there not a test that needs updating?

**Pull request merged**